reports/blueprint/report.tex

```diff
@@ -46,10 +46,29 @@
 
     \end{center}
 \end{titlepage}
-\section*{Executive Summary}
-% pop a one pager in here. Ideally less
-% Timothy
-\newpage
+\section{Executive Summary}
+
+This report outlines TeachEE, a device which facilitates online learning for
+electrical engineering labs. It describes the project's scope: a minimum viable
+product for the purpose of this capstone project, with room for scope expansion
+later in the year if circumstances allow it. The minimum viable product consists
+of a oscilloscope, desktop app, and communication between a computer and the
+device via USB. As a general rule, as long as waveforms can be produced in the
+hardware and displayed from software, a minimal product is satisfied.
+
+It also outlines a road-map for this project, and that we have so far completed
+most hardware related tasks from which we can now develop the software. Overall,
+it reports that the project is in a healthy state, showing progress in all
```

Comment:
Maybe just say project is on schedule.

Reply:
ced8fb282275d20249288b0514e85b99a7f5d580